sites/public/src/pages/create-account.tsx

```diff
@@ -72,182 +73,161 @@ export default () => {
+                required={true}
+                error={errors.dob}
+                name="dob"
+                id="dob"
+                watch={watch}
+                validateAge18={true}
+                errorMessage={t("errors.dateOfBirthErrorAge")}
+                label={t("application.name.yourDateOfBirth")}
+              />
+              <p className={"field-sub-note"}>{t("application.name.dobHelper")}</p>
+            </CardSection>
 
-          <div className="form-card__group border-b">
-            <Field
-              caps={true}
-              type="password"
-              name="password"
-              note={t("authentication.createAccount.passwordInfo")}
-              label={t("authentication.createAccount.password")}
-              placeholder={t("authentication.createAccount.mustBe8Chars")}
-              validation={{
-                required: true,
-                minLength: 8,
-                pattern: passwordRegex,
-              }}
-              error={errors.password}
-              errorMessage={t("authentication.signIn.passwordError")}
-              register={register}
-            />
-            <p className="text text-gray-750 text-sm pb-2">
-              {t("authentication.createAccount.reEnterPassword")}
-            </p>
-            <Field
-              type="password"
-              name="passwordConfirmation"
-              placeholder={t("authentication.createAccount.mustBe8Chars")}
-              validation={{
-                validate: (value) =>
-                  value === password.current ||
-                  t("authentication.createAccount.errors.passwordMismatch"),
-              }}
-              onPaste={(e) => {
-                e.preventDefault()
-                e.nativeEvent.stopImmediatePropagation()
-                return false
-              }}
-              onDrop={(e) => {
-                e.preventDefault()
-                e.nativeEvent.stopImmediatePropagation()
-                return false
-              }}
-              error={errors.passwordConfirmation}
-              errorMessage={t("authentication.createAccount.errors.passwordMismatch")}
-              register={register}
-              label={t("authentication.createAccount.reEnterPassword")}
-              readerOnly
-            />
-
-            <div className="text-center mt-10">
+            <CardSection divider={"inset"} className="mx-12 p-0 py-8">
+              <Field
+                caps={true}
+                type="email"
+                name="email"
+                label={t("application.name.yourEmailAddress")}
+                validation={{ required: true, pattern: emailRegex }}
+                error={errors.email}
+                errorMessage={t("authentication.signIn.loginError")}
+                register={register}
+                controlClassName={styles["create-account-input"]}
+              />
+            </CardSection>
+            <CardSection className="mx-12 p-0 py-8 space-y-3">
```

Comment:
I wouldn't expect to see any class names for spacing added onto the Card components - we should be able to take whatever is in seeds.